packages/web-components/src/utils/utils.spec.ts

```diff
@@ -14,6 +14,139 @@ describe('format', () => {
   });
 
   it('formats first, middle and last names', () => {
-    expect(format('Joseph', 'Quincy', 'Publique')).toEqual('Joseph Quincy Publique');
+    expect(format('Joseph', 'Quincy', 'Publique')).toEqual(
+      'Joseph Quincy Publique',
+    );
+  });
+});
+
+describe('getSlottedNodes()', () => {
+  it('returns an array of nodes from a custom element in the shadow dom', async () => {
+    // Shadow DOM Custom Element
+    class CustomElement extends window.HTMLElement {
+      constructor() {
+        super();
+        this.attachShadow({ mode: 'open' });
+        this.shadowRoot.innerHTML = `<h6>ABC</h6><div>DEF</div><p>GHI</p>`;
+      }
+    }
+
+    window.customElements.define('custom-element', CustomElement);
+
+    const defaultElement = document.createElement('custom-element');
+    const slottedNodes = getSlottedNodes(defaultElement, null);
+    expect(await slottedNodes[2].nodeName).toEqual('P');
```

Comment:
`getSlottedNodes` isn't returning a Promise, so I don't think we need any of these `await`s.